src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -279,7 +279,8 @@ HttpStatusCode mongoQueryContext
   std::map<std::string, std::string>&  uriParams,
   std::map<std::string, bool>&         options,
   long long*                           countP,
-  const std::string&                   apiVersion
+  const std::string&                   apiVersion,
+  StringFilter*                        stringFilterP
```

Comment:
If we move StringFilter to Scope, we don't need this new paramter (it will come "inside" `QueryContextRequest* requestP`).